### PR TITLE
MAT-7298: add publisher to main library

### PR DESCRIPTION
**Pull request by chubert-sb**
## MADiE FHIR SERVICE

Jira Ticket: [MAT-7298](https://jira.cms.gov/browse/MAT-7298)
(Optional) Related Tickets:

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/madiefhirservice/services/MeasureBundleService.java [hunk 1]
```diff
@@ -109,6 +109,7 @@ public List<Bundle.BundleEntryComponent> createBundleComponentsForLibrariesOfMad
       final String accessToken) {
     Library library =
         getMeasureLibraryResourceForMadieMeasure(expressions, madieMeasure, accessToken);
+    library.setPublisher(madieMeasure.getMeasureMetaData().getSteward().getName());
```
Comment (nmorasb) [inline src/main/java/gov/cms/madie/madiefhirservice/services/MeasureBundleService.java]: does this blow up if someone creates a measure, adds CQL, adds a test case, and then tries to execute it? (thinking about the executable measure bundle)

Comment (chubert-sb) [reply src/main/java/gov/cms/madie/madiefhirservice/services/MeasureBundleService.java]: added a null check for this case

**Review by nmorasb — APPROVED** [review]
